base_station/cameras/README.md

```diff
@@ -0,0 +1,18 @@
+Code to control the USB Cameras
```

Comment:
I don't know how this is here? On my computer I deleted all of base_station/cameras...

Reply:
ok seems like git is being weird... the folder doesn't actually exist i think

Reply:
Yeah I feel like I've removed things multiple times at this point...

Reply:
I could be imagining it though